context/app/static/js/components/Detail/entityHeader/EntityHeaderContent/EntityHeaderContent.jsx

```diff
@@ -26,7 +26,9 @@ function EntityHeaderContent({ display_doi, entity_type, data, summaryInView })
         <AnimatedFlexContainer style={props} key={key}>
           {iconMap[entity_type]}
           <EntityHeaderItem text={display_doi} />
-          {data.map((d) => d && <EntityHeaderItem text={d} key={d} />)}
+          {Object.entries(data).map(([k, v]) => (
+            <EntityHeaderItem text={v.value ? v.value : `undefined ${v.label}`} key={k} />
```

Comment:
- Could`v.value` be zero? (... thought, at least for our current data, probably not)
- Use `||`:
```suggestion
            <EntityHeaderItem text={v.value || `undefined ${v.label}`} key={k} />
```